integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/passthru/transport/test/ESBJAVA3336HostHeaderValuePortCheckTestCase.java

```diff
@@ -103,18 +103,17 @@ private void applyProperty(File srcFile, String key, String value) throws IOExce
             fos = new FileOutputStream(destinationFile);
             Properties properties = new Properties();
             properties.load(fis);
+            fis.close();
```

Comment:
Shouldn't these (clearing up resources) go to finally block?

Reply:
We have to close the FileInputStream before accessing the FileOutputStream when running the test in Windows environment. Because both the FileInputStream and FileOutputStream  accessing the same file.

Reply:
What will happen if an exception is thrown before that? FileInputStream never closed?

Reply:
Fixed. Please refer latest commit.